src/common/runtime.js

```diff
@@ -84,7 +84,7 @@ export const never/*:Task<Never, any>*/ =
 
 export const respond = /*::<message>*/
   (message/*:message*/)/*:Task<Never, message>*/ =>
-  Task.future(() => Promise.resolve(message));
+  Task.create(resolve => resolve(message));
```

Comment:
We need to keep a promise wrapper here in order to keep an async resolution behavior as it was prior to this change (See https://github.com/browserhtml/browserhtml/pull/1003#issuecomment-211648650 for more details)